WordPress/Classes/Models/Media.h

```diff
@@ -52,6 +53,7 @@ typedef NS_ENUM(NSUInteger, MediaType) {
 
 @property (nonatomic, strong) Blog *blog;
 @property (nonatomic, strong, nullable) NSSet *posts;
+@property (nonatomic, strong, nullable) NSSet *featuredOn;
```

Comment:
I'd be tempted to rename this to `featuredOnPosts` just to make its use a little clearer.